app/lib/bk/compat/parsers/gha/steps.rb

```diff
@@ -32,7 +32,17 @@ def translate_run(step)
       end
 
       def translate_uses(step)
-        "# action #{step['uses']} can not be translated just yet"
+        case step['uses']
+        when /\Aactions\/setup-python@v\d+\z/
+          python_version = step.dig('with', 'python-version') || 'latest'
+          image_string = "python: #{python_version}"
+          BK::Compat::Plugin.new(
+            name: 'docker',
+            config: {
+              'image' => image_string
+            }
+          )
+        end
```

Comment:
Going off the errors in the build from the snapshots, I think this might need an `else nil` type deal. and then an update to the snapshots and its looking good i think

Reply:
Good call! Added that support in.

Reply:
Ah, when is has a `uses` that doesn't match it inserts an empty line...

Reply:
Implemented a fix for that but need to assess the output; if we want it to match out "expected" then it'll need some work.

Reply:
I pushed a change to fix the error and updated the snapshots. @mcncl i think its good now 👍